browser/css/jsdialogs.css

```diff
@@ -1746,7 +1756,7 @@ input[type='number']:hover::-webkit-outer-spin-button {
 }
 
 .notebookbar .ui-content.unobutton:hover,
-.jsdialog .ui-content.unobutton:hover {
+.jsdialog .ui-content.unobutton:hover .ui-combobox-button:hover{
```

Comment:
This seems to be a mistake or maybe you meant to do something else (?) because as far as I recall we do not have combo-boxes inside of unobutton